modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upgrade/registry/DDMFormWebUpgradeStepRegistrator.java

```diff
@@ -27,6 +27,11 @@ public void register(Registry registry) {
 			"0.0.1", "1.0.0", new UpgradeDDMFormAdminPortletId(),
 			new UpgradeDDMFormPortletId(),
 			new UpgradeDDMFormPortletPreferences());
+
+		registry.register(
+			"1.0.0", "1.1.0",
```

Comment:
@p-albuquerque why did you decide to make a minor change instead of a micro (see [documentation](https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Schema-version))?

Reply:
@carolmariaabb  I had read this doc, but now I realize I missunderstand the MINOR part "They are required for the application to work (it will fail without applying the data changes)". Now I believe it fits more with a MICRO change.